source/extensions/extensions_build_config.bzl

```diff
@@ -0,0 +1,5 @@
+# FIXFIX comments/docs
+EXTENSIONS = {
+    # FIXFIX comments
+    "envoy.filters.http.lua": "@envoy//source/extensions/filters/http/lua:config",
```

Comment:
Will all the filters apart from HCM and TCP proxy end up here essentially?

Reply:
Yes. This is not necessarily true for tests right now, but I will be adding more detailed comments in all of this.